lua/lspconfig/sqls.lua

```diff
@@ -0,0 +1,31 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local root_pattern = util.root_pattern(".git", "config.yml")
+
+configs.sqls = {
+  default_config = {
+    cmd = {"sqls", "-config", "$HOME/.config/sqls/config.yml"};
```

Comment:
Is the `-config` flag necessary here? According to the README, `sqls` reads from `$XDG_CONFIG_HOME` automatically: https://github.com/lighttiger2505/sqls#configuration-methods

Reply:
Agree, the default config should not setup config.yml to a specific folder. It was updated in latest push